package.json

```diff
@@ -16,14 +16,21 @@
     "url": "git@github.com:ca-la/react-portal.git"
   },
   "author": "CALA <devops@ca.la>",
-  "license": "UNLICENSED",
+  "license": "MIT",
   "bugs": {
     "url": "https://github.com/ca-la/react-portal/issues"
   },
   "homepage": "https://github.com/ca-la/react-portal#readme",
   "peerDependencies": {
     "react": "^16.8.5"
   },
+  "keywords": [
+    "React",
+    "TypeScript",
+    "react-portal",
+    "react-native-portal",
+    "portal"
```

Comment:
Any reason not to add `pixi` here?

Reply:
Sure I guess we could do that! It's not necessarily related to the pixi project, so I'll add in `pixi-react`?

Reply:
eca67bd